nowcasting_dataset/utils.py

```diff
@@ -196,3 +196,36 @@ def get_start_and_end_example_index(batch_idx: int, batch_size: int) -> (int, in
     end_example_idx = (batch_idx + 1) * batch_size
 
     return start_example_idx, end_example_idx
+
+
+class DummyExecutor(futures.Executor):
```

Comment:
do we need this? or was this just for debugging?

Reply:
Good question!

It's not used in the code right now.  But I did find it super-helpful for debugging.  If we didn't put it here, where should we put it?  I suppose we could have a `nowcasting_debugging` repo :slightly_smiling_face:

Reply:
hmm, yea im not sure - perhaps easiest just to leave it for the moment with a comment (as you have done)